lib/logic/player/music_player.dart

```diff
@@ -667,7 +664,6 @@ class MusicPlayer extends AudioPlayer {
         // artDownscaleHeight,
         fastForwardInterval: const Duration(seconds: 5),
         rewindInterval: const Duration(seconds: 5),
-        androidEnableQueue: true,
```

Comment:
Is this now enabled by default?

Reply:
Yes. see [this warning in their code](https://github.com/ryanheise/audio_service/blob/60c6dfd24d2b571fe0a0c77fbb08daeba259b33e/audio_service/lib/audio_service.dart#L1359):
```
print('NOTE: androidEnableQueue is always true from 0.18.0 onwards.');
```